apps/fluent-tester/src/TestComponents/Button/E2EButtonTest.tsx

```diff
@@ -60,6 +62,14 @@ export const E2EButtonExperimentalTest: React.FunctionComponent = () => {
         >
           Press &quot;a&quot; or &quot;b&quot; after focusing this button
         </Button>
+        <Button
+          testID={BUTTON_FOCUSABLE_TEST_COMPONENT}
+          onClick={onClick}
+          accessibilityLabel={BUTTON_FOCUSABLE_TEST_COMPONENT_LABEL}
+          focusable={false}
+        >
+          This button isn&apos;t focusable (but isn&apos;t disabled or indeterminate either)
```

Comment:
Is the "&apos;" required? I don't think I've seen it before

Reply:
I guess it's right up above in a different button so uh, sure

Reply:
you mean `isn't` directly without escaping it? VSCode seemed to complain about it, I didn't actually try building with that

Reply:
that's so strange, so VSCode is picking up some eslint rule from somewhere, but `npm run build` does not complain.

![image](https://user-images.githubusercontent.com/72474613/182460425-19efbaad-be63-4136-88dc-105b0060e292.png)

![image](https://user-images.githubusercontent.com/72474613/182460496-d8f88053-f737-461d-9604-4172fa8ede7f.png)

I'll guess I'll leave it escaped?

Reply:
Yeah that's fine, I was just curious.